wireless_modules/common_sensors/dht_sensor.py

```diff
@@ -0,0 +1,40 @@
+import dht
+
+
+class DhtSensor:
+    def __init__(self, pin, queries=1):
+        """
+        Initialise the DHT sensor for temperature and humidity readings
+        :param pin: A Pin object
+        :param queries: Number of times the sensor will be read every 2 seconds
```

Comment:
Hahah funny that in my last PR I had issues with over-commenting and in this one I'm missing key comments. 
Anyways, I added class comments just before the `_init_` so that should help explain it I hope